doc/design.md

```diff
@@ -231,6 +231,30 @@ Covers:
 
 Needs: impl, utest, itest
 
+## Administration Convenience Scripts
+ 
+RLS adapter provides scripts which make administration of RLS more user-friendly.
+
+### Add a new role
+
+`ADD_RLS_ROLE(role_name, role_id)` allows administrators to add a new RLS role to the table named `EXA_ROLES_MAPPING`.
+If table already exists, it adds a role to the existing table. If not it creates a new table and adds a role.
```

Comment:
Those are individual design requirements. It's easy to tell since each needs at least one test.

Reply:
Suggestion:
> `ADD_RLS_ROLES` creates the table `EXA_ROLES_MAPPING` if it does not exist.

Reply:
Fixed